etc/cc-dev.conf

```diff
@@ -49,9 +41,26 @@ server {
     proxy_pass         http://localhost:7782/connect;
     proxy_redirect     off;
 
-    proxy_set_header   Host             $host;
-    proxy_set_header   X-Real-IP        $remote_addr;
-    proxy_set_header   X-Forwarded-For  $proxy_add_x_forwarded_for;
+    proxy_max_temp_file_size 0;
+
+    client_max_body_size       10m;
+    client_body_buffer_size    128k;
+
+    proxy_connect_timeout      90;
+    proxy_send_timeout         90;
+    proxy_read_timeout         90;
```

Comment:
It looks like nginx can take a units suffix (at least for seconds).  If that's correct, lets write all these timeouts as as `90s` (or whatever value is chosen) for clarity.

Reply:
Done.